### PR TITLE
Add patch release note for v1.12.7, 1.13.5, 1.14.5, 1.15.2

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add patch release note for v1.12.7, 1.13.5, 1.14.5, 1.15.2, four releases.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.14.md [hunk 1]
```diff
@@ -23,6 +27,20 @@
         * [Other Notable Changes](#other-notable-changes)
         * [Bug Fixes](#bug-fixes)
 
+# v1.14.5
+
+## Downloads for v1.14.5
+
+Download v1.14.4 in the [v1.14.5 release page](https://github.com/kubeedge/kubeedge/releases/tag/v1.14.5).
```
Comment (Shelley-BaoYue) [inline CHANGELOG/CHANGELOG-1.14.md:34]:
```suggestion
Download v1.14.5 in the [v1.14.5 release page](https://github.com/kubeedge/kubeedge/releases/tag/v1.14.5).
```

File: CHANGELOG/CHANGELOG-1.15.md [hunk 2]
```diff
@@ -9,6 +13,22 @@
         * [Important Steps before Upgrading](#important-steps-before-upgrading)
 
 
+# v1.15.2
+
+## Downloads for v1.15.2
+
+Download v1.15.1 in the [v1.15.2 release page](https://github.com/kubeedge/kubeedge/releases/tag/v1.15.2).
```
Comment (Shelley-BaoYue) [inline CHANGELOG/CHANGELOG-1.15.md:20]:
```suggestion
Download v1.15.2 in the [v1.15.2 release page](https://github.com/kubeedge/kubeedge/releases/tag/v1.15.2).
```

File: CHANGELOG/CHANGELOG-1.14.md (hunk 1 above)
Comment (fisherxu) [reply CHANGELOG/CHANGELOG-1.14.md]: Done.

File: CHANGELOG/CHANGELOG-1.15.md (hunk 2 above)
Comment (fisherxu) [reply CHANGELOG/CHANGELOG-1.15.md]: Done.

**Review by Shelley-BaoYue — COMMENTED** [review]
/approve

**Pull request merged**